### PR TITLE
add rss feed to replies #111

### PR DESCRIPTION
**Pull request by akosourov**
This feature adds /rss/reply endpoint that generates RSS feed with replies for the comment. 
Replies are sorted by time from last to first. Amount of replies is limited up to maxRssItems (20).
For sorting purpose sortComments function from engine package had to do public, so I'm not sure that it is a good solution
@umputun I'm will be glad any feedback

### CODE REVIEW

File: backend/app/rest/api/rss.go [hunk 1]
```diff
@@ -88,6 +90,48 @@ func (s *Rest) rssSiteCommentsCtrl(w http.ResponseWriter, r *http.Request) {
...
+	})
+
+	if err != nil {
+		rest.SendErrorJSON(w, r, http.StatusBadRequest, err, "can't find replies")
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/xml; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	if _, err := w.Write([]byte(data)); err != nil {
```
Comment (golangci) [inline backend/app/rest/api/rss.go]: unnecessary conversion

**Comment by akosourov** [comment]
This relates to #111 in more general way. If you want to be notified when someone replied to your comment or replied to thread (comment) that you are interested in - you can subscribe to it through RSS reader.
This was the goal

**Comment by umputun** [comment]
I read #111 differently. From what I see user would like to have a single "replies to all of my comments" RSS instead or resubscribing every time he commented on a different (sub)tree. I see how `/reply?user_id&site_id` could be useful here, but don't have in mind an easy way to implement it. Maybe you can try to address it in three steps:
1. get last user's comments
1. apply `findReplies` to each one. 
1. sort and remove duplicates

**Comment by umputun** [comment]
pls, note - I'm not saying your version of `/rss/reply` is useless, just can't imagine the practical use case. Another thing is how to integrate it into UI. In order to expose this "per-comment" RSS each and every comment should have some "subscribe" link, which feels excessive to me especially for such exotic technology as RSS ;)

**Comment by akosourov** [comment]
Ok, I agree. If you don't mind I'll try to implement `/reply?user_id&site_id` solution and come back later :)

**Comment by umputun** [comment]
sure, thx

**Pull request closed without merging**